### DISCUSSION TITLE
Copilot completions and chat is not work at Jetbrains Idea

### DISCUSSION THREAD

**Discussion by Qoozm** [community/community#129043] [Copilot Conversations]
### Select Topic Area

Question

### Body

Chat is always loading
<img width="461" alt="image" src="https://github.com/community/community/assets/29374874/7c6cad47-2e46-44b9-b859-c937375d8e8c">

i try to back old version, but it doesn't seems to work

current plugin version: 1.5.9.5799

**Comment by Qoozm** [comment]
i hava fund the problem, it's my network limit, set a proxy in idea is ok

**Comment by Akash1134** [comment]
Hi @Qoozm ,

Few questions to better diagnose the issue: 
- Are you using Copilot in a proxy environment? 
- Can you please ensure your IntelliJ IDE and Copilot extensions are fully up to date.
- Have you checked if the 'Use Editor Networking' option within your Copilot settings is enabled?
 
This may help with your issue --> please perform the diagnostics outlined in [Troubleshooting network errors for GitHub Copilot](https://docs.github.com/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot#title-h1) in the GitHub Docs and let us know how it goes. 

Thanks!

**Reply by Qoozm** [reply]
thank for your reply

- i am using at company network
- i'm sure IntelliJ IDE and Copilot extensions are up to date
- i try this option, but useless

according to [Configuring network settings for GitHub Copilot](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-network-settings-for-github-copilot), i set my own proxy server in idea, and now it's running.

i think it's my company firewall limited, but i'm not sure, i can't see the blacklist
